tests/robustness/snapmeta/kopia_snapshotter.go

```diff
@@ -178,3 +180,23 @@ func (ks *KopiaSnapshotter) ConnectOrCreateFilesystemWithServer(serverAddr, repo
 func (ks *KopiaSnapshotter) Cleanup() {
 	ks.snap.Cleanup()
 }
+
+// GetRepositoryStatus returns the repository status in JSON format.
+func (ks *KopiaSnapshotter) GetRepositoryStatus() (cli.RepositoryStatus, error) {
+	a1, _, _ := ks.snap.Run("repository", "status", "--json")
```

Comment:
Should we check for error here?
```suggestion
	a1, _, err := ks.snap.Run("repository", "status", "--json")
	if err != nil {
	  return nil, err
	}
```